.github/workflows/build_source_release.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Source Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
```

Comment:
Could we just add this as a second job called "build-source-release" to the existing `build_release.yml`?

The jobs will run in parallel, and will ensure that they both get triggered on the same events.

Reply:
i was looking into that earlier, but i'm not familiar with github workflows and shoehorning it into the existing script seemed to involve either arcane knowledge or adding a lot of if: special cases to the existing targets. having it as a separate task seems to be much cleaner, and allowed me to come up with a solution to begin with. both tasks do actually run in parallel in my tests, and the build-source-release tasked finished much faster (naturally, as it doesn't need to compile C++ code...)

Reply:
OK we can leave it separate for now